src/autopas/containers/linkedCells/traversals/LCC08CellHandlerUtility.cpp

```diff
@@ -0,0 +1,189 @@
+/**
+ * @file LCC08CellHandlerUtility.cpp
+ * @author J. Schuhmacher
+ * @date 11.07.2024
+ */
+
+#include "LCC08CellHandlerUtility.h"
+
+namespace autopas::internal {
+
+constexpr int calculateCell2Index(int overlap1, const C08CellDirection &direction, int z) {
+  switch (direction) {
+    case C08CellDirection::FRONT_LEFT:
+      return z;
+    case C08CellDirection::BACK_LEFT:
+      return overlap1 * (overlap1 - 1) + z;
+    case C08CellDirection::FRONT_RIGHT:
+      return overlap1 * overlap1 * (overlap1 - 1) + z;
+    case C08CellDirection::BACK_RIGHT:
+      return overlap1 * overlap1 * overlap1 - overlap1 + z;
+  }
+  return 0;
+}
+
+constexpr std::pair<int, int> toDistVectorMultiplier(const C08CellDirection &direction) {
+  switch (direction) {
+    case C08CellDirection::FRONT_LEFT:
+      return std::make_pair(0, 0);
+    case C08CellDirection::BACK_LEFT:
+      return std::make_pair(0, 1);
+    case C08CellDirection::FRONT_RIGHT:
+      return std::make_pair(1, 0);
+    case C08CellDirection::BACK_RIGHT:
+      return std::make_pair(1, 1);
+  }
+  return std::make_pair(0, 0);
+}
+
+constexpr bool includeC08Case(const C08CellDirection &direction, const std::array<int, 3> &overlap, int x, int y,
+                              int z) {
+  switch (direction) {
+    case C08CellDirection::BACK_LEFT:
+      return y != overlap[1] and z != 0;
+    case C08CellDirection::FRONT_RIGHT:
+      return x != overlap[0] and (y != 0 or z != 0);
+    case C08CellDirection::BACK_RIGHT:
+      return y != overlap[1] and x != overlap[0] and z != 0;
+    case C08CellDirection::FRONT_LEFT:
+    default:
+      return true;
+  }
+}
+
+std::vector<int> computeCellOffsetsC08(const std::array<unsigned long, 3> &cellsPerDimension,
+                                       const std::array<int, 3> &overlap) {
+  using utils::ArrayMath::prod, utils::ArrayMath::addScalar;
```

Comment:
```suggestion
  using utils::ArrayMath::addScalar;
  using utils::ArrayMath::prod;
```

No real difference just personal preference :shrug: